libs/SmartStore/SmartStore/Classes/SFSmartStore.m

```diff
@@ -1351,6 +1341,19 @@ - (void)registerSoupWithSpec:(SFSoupSpec*)soupSpec withIndexSpecs:(NSArray*)inde
     }
     [self insertIntoSoupIndexMap:soupIndexMapInserts withDb:db];
     
+    // LogEvent
+    NSMutableArray<NSString *> *features = [[NSMutableArray alloc] init];
+    if (soupUsesJSON1) {
+        [features addObject:@"JSON1"];
+    }
+    if (soupUsesExternalStorage) {
+        [features addObject:@"ExternalStorage"];
+    }
+    if ([self hasFts:soupSpec.soupName withDb:db]) {
```

Comment:
I did think of it.  I guess the logeevent should move then too right.
``` 
[[self class] logAnalyticsEventWithName:@"registerSoup" userAccount:self.user storeAttributes:nil features:features];
```
 The question then is if we should persist the event after completion of registration. Additionally  event logging code in one continuos code block vs. distributed over.